.github/workflows/example.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           insecure_skip_tls_verify: true
           namespace: ${{ env.TEST_NAMESPACE }}
 
+      # This step should create a deployment, service, and route to run your app and expose it to the internet.
```

Comment:
let's say "This step _will_  create" so it doesn't sound like we're unsure about whether or not our actions work. lol